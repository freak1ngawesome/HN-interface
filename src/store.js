import { configureStore } from '@reduxjs/toolkit'
import { setupListeners } from '@reduxjs/toolkit/query'
import { newsApi } from './slices/apiSlice'

export const store = configureStore({
  reducer: {
    [newsApi.reducerPath]: newsApi.reducer,
  },
  middleware: (getDefaultMiddleware) =>
    getDefaultMiddleware().concat(newsApi.middleware),
})

setupListeners(store.dispatch)
