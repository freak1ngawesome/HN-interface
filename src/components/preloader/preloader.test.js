import React from 'react';
import Preloader from './preloader';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import store from '../../store';

describe('Header tests',() => {
  const preload = renderer.create(<Preloader store={store}/>);

  it('dad',() => {
    expect(preload.toJSON()).toMatchSnapshot()
  })

});