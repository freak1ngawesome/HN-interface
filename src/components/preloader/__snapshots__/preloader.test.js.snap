// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests dad 1`] = `
<div
  className="preloader"
  id="preloader"
>
  <img
    alt="loading..."
    className="preloader__img"
    src="preloader.svg"
  />
</div>
`;
